arch/arm/core/fatal.c

```diff
@@ -101,15 +101,39 @@ void z_arm_fatal_error(unsigned int reason, const struct arch_esf *esf)
  *
  * @param esf exception frame
  * @param callee_regs Callee-saved registers (R4-R11)
+ * @param exc_return EXC_RETURN value present in LR after exception entry.
  */
-void z_do_kernel_oops(const struct arch_esf *esf, _callee_saved_t *callee_regs)
+void z_do_kernel_oops(const struct arch_esf *esf, _callee_saved_t *callee_regs, uint32_t exc_return)
```

Comment:
In #76221 I [made sure](https://github.com/zephyrproject-rtos/zephyr/pull/76221/files#diff-81474c76b65788e3a74efda285096d1f735f501b8f898fc0892e9a6276e823b6R344-R346) Cortex-A/Cortex-R don't pass garbage here, but I don't know if that's really necessary.

Reply:
Makes sense to me